### PR TITLE
support consumer pending counts

### PR DESCRIPTION
**Pull request by ripienaar**
Signed-off-by: R.I.Pienaar <rip@devco.net>

### CODE REVIEW

File: jetstream.go [hunk 1]
```diff
@@ -97,6 +99,14 @@ func (m *Msg) parseJSMsgMetadata() (*JetStreamMsgMetaData, error) {
 	}
 	meta.TimeStamp = time.Unix(0, int64(tsi))
 
+	meta.Pending = -1
```
Comment (derekcollison) [inline jetstream.go:102]: We should do scanf like server, I think that will be generally faster and I may do a very low level fast version which we could borrow here but for now I would do Sscanf like server.

**Review by derekcollison — CHANGES_REQUESTED** [review]

File: jetstream.go (hunk 1 above)
Comment (ripienaar) [reply jetstream.go:102]: yeah I spent a bunch of time wasting on Sscanf and its no good - in server you know the Stream and Consumer, here we have to also extract the Stream and Consumer.  fmt.Sscanf only works with strings if they are space seperated not `.` as here.

Comment (derekcollison) [reply jetstream.go:102]: ok so keep as is and I can do a high speed one at some point in Go and you can take it too.

**Review by derekcollison — APPROVED** [review]
LGTM

**Pull request merged**